blueocean-commons/src/main/java/io/jenkins/blueocean/commons/MapsHelper.java

```diff
@@ -0,0 +1,66 @@
+package io.jenkins.blueocean.commons;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Some helpers to replace some Guava collections tools
+ */
+public class MapsHelper {
+    private MapsHelper() {
+    }
+
+    public static <K,V> Map<K,V> of(Map<K,V> m) {
+        Map<K,V> map = new HashMap<>(m);
+        return Collections.unmodifiableMap(map);
+    }
+
+    public static <K,V> Map<K,V> of(K k1, V v1) {
+        Map<K,V> map = new HashMap<>();
+        map.put(k1, v1);
+        return Collections.unmodifiableMap(map);
```

Comment:
```suggestion
        return Collections.singletonMap(k1,v1);
```
https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singletonMap-K-V-